src/axom/primal/operators/detail/winding_number_2d_memoization.hpp

```diff
@@ -184,6 +238,13 @@ class NURBSCurveGWNCache
   mutable axom::Array<std::map<std::pair<int, int>, BezierCurveData<T>>> m_bezierSubdivisionMaps;
```

Comment:
Is some other object hanging onto a const reference or pointer to this cache, making it necessary to mark `getSubdivisionData` `const`, leading to this `mutable`?